tests/acceptance/install-test-slow.js

```diff
@@ -7,32 +7,29 @@ var assertFile = require('../helpers/assert-file');
 var conf       = require('../helpers/conf');
 var ember      = require('../helpers/ember');
 var path       = require('path');
-var remove     = Promise.denodeify(require('fs-extra').remove);
+var remove     = Promise.denodeify(require('rimraf'));
 var root       = process.cwd();
 var tmp        = require('tmp-sync');
 var tmproot    = path.join(root, 'tmp');
 var expect     = require('chai').expect;
 
 describe('Acceptance: ember install', function() {
-  this.timeout(30000);
+  this.timeout(40000);
   var tmpdir;
 
   before(function() {
     conf.setup();
+    tmpdir = tmp.in(tmproot);
+    process.chdir(tmpdir);
   });
 
   after(function() {
     conf.restore();
-  });
-
-  beforeEach(function() {
-    tmpdir = tmp.in(tmproot);
-    process.chdir(tmpdir);
+    process.chdir(root);
   });
 
   afterEach(function() {
-    process.chdir(root);
-    return remove(tmproot);
+    return remove(tmproot, { maxBusyTries: 10 });
```

Comment:
Well actually, with the maxBusyTries prop set to 10 it times out, when the prop is not set (defaults to 3) it returns the busy signal error.